packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -265,17 +287,42 @@ export class GarbageCollector implements IGarbageCollector {
         this.deleteTimeoutMs = this.gcOptions.deleteTimeoutMs ?? defaultDeleteTimeoutMs;
 
         let prevSummaryGCVersion: number | undefined;
+
         // GC can only be enabled during creation. After that, it can never be enabled again. So, for existing
         // documents, we get this information from the metadata blob.
         if (existing) {
             prevSummaryGCVersion = getGCVersion(metadata);
             // Existing documents which did not have metadata blob or had GC disabled have version as 0. For all
             // other exsiting documents, GC is enabled.
             this.gcEnabled = prevSummaryGCVersion > 0;
+
+            if (metadata?.sessionTimeoutMs !== undefined) {
```

Comment:
This if condition is unnecessary. You can do `this.sessionExpiryMs = metadata?.sessionTimeoutMs;`